plugins/accelerated-peft/tests/test_peft_plugins.py

```diff
@@ -86,6 +89,45 @@ def test_configure_gptq_plugin():
 
         e.match(f"AutoGPTQAccelerationPlugin: Value at '{key}'")
 
+def test_autogptq_loading():
+    def autogptq_unavailable(package_name: str):
+        return False
+
+    # - Test that error is thrown when use_external_lib is True but no package found.
+    with pytest.raises(
+        AssertionError,
+        match = "Unable to use external library, auto_gptq module not found. \
+                Refer to README for installation instructions \
+                as a specific version might be required."
+    ):
+        with patch(
+                    "transformers.utils.import_utils."
+                    "_is_package_available",
+                    autogptq_unavailable,
+        ):
+            with instantiate_framework(
+                update_configuration_contents(
+                    read_configuration(CONFIG_PATH_AUTO_GPTQ),
+                    "peft.quantization.auto_gptq.use_external_lib",
+                    True,
+                ),
+                require_packages_check=False,
+            ) as framework:
+                pass
+
+    # - Test that gptqmodel is used when use_external_lib is False
```

Comment:
have an extension of this test where
- we load `CONFIG_PATH_AUTO_GPTQ`  and delete away the key
- it will detect it as False
- and it will not use our gptq